src/labels/CustomLabels.labels

```diff
@@ -1640,6 +1648,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>The total Amount of all Closed Won Opps is not equal to the RD Amount.</shortDescription>
         <value>The combined Amount of all related Closed Won and Pledged Opportunities must match the Amount of a Fixed Length Recurring Donation.</value>
     </labels>
+    <labels>
+        <fullName>RD_ErrorRD2BatchJobCannotBeRun</fullName>
```

Comment:
Can we name custom labels to indicate requirement and/or replace RD1 with legacy and RD2 with enhance? For example, `_EnhancedBatchCannotBeRun` or `_EnhancedBatchRequiresEnablement`. Similarly, `_LegacyBatchCannotBeRun`.

Which prefix should these custom labels use? The enhanced RD batch custom label has `RD_` prefix while the legacy batch custom label has `RD2_` prefix.

Reply:
Good point. Label names updated
I do think the prefixes are correct. The RD2 prefixed error is used in the RD2 job, while the RD prefixed error is used in the RD job.